### PR TITLE
feat-0T170-28-create-migration-and-model-of-slides

### PR DESCRIPTION
**Pull request by fabiianZ**
**Crear migración y modelo de Slides**
COMO usuario administrador
QUIERO gestionar las imágenes
PARA modificar eficientemente el contenido visual

Criterios de aceptación:
Los Slides de la pantalla de inicio serán gestionados de forma dinámica por el administrador del sitio. Los mismos tendrán como campos imageUrl, text, order y organizationId (ya que pertenecerán una ong)

------------------------

the migration of AddOrganizationidToSlides organization:references, is commented out because until the merge is done, it does not allow the db to migrate. and with respect to belongs to in the slide model, it is also commented out to avoid errors

### CODE REVIEW

**Review by FernandoCiballero — APPROVED** [review]
Great Job Fabian!! 🚀

**Review by DevChristians — APPROVED** [review]

File: spec/factories/slides.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :slide do
+    imageUrl { nil }
+    text { "MyText" }
+    order { 1 }
```
Comment (Ezequiel-Padilla) [inline spec/factories/slides.rb:5]: Use Faker for this

File: app/models/slide.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Slide < ApplicationRecord
+  #belongs_to :organization
+  has_one_attached :imageUrl
+  validates :text, presence: true
+  validates :order, presence: true
```
Comment (Ezequiel-Padilla) [inline app/models/slide.rb:5]: add an empty line between validates and active storage

Comment (fabiianZ) [reply app/models/slide.rb:5]: Done

**Review by FernandoCiballero — APPROVED** [review]
Great Job Fabian!!! 👍  I've added some missing files. Please don't forget to check the syntax offences with `bundle exec rubocop` every time you create something new in the project.

**Pull request merged**